WordPress/gutenberg
# [Block: Post comment author]: Add link settings and block supports

## Description

Closes #30572.

Used #30573 as a reference.

## How has this been tested?




1. Add a post comment block (using an existing comment ID)
2. Add an inner post comment author block
3. Confirm that the color and typography options work
4. Confirm that the link option works.
5. Confirm that the target blank/self works.


## Screenshots 

Loom with a summary:
https://www.loom.com/share/924ffd9e1d9d469ea4bbba6fe6529219

Source code generated on the frontend:
![comment_author_frontend_code](https://user-images.githubusercontent.com/37012961/137315745-bd651a67-606d-4a4a-944f-656f778150bf.png)

## Types of changes
Enhancement

## Checklist:
